cache/cache.go

```diff
@@ -68,13 +70,13 @@ func RebuildCmd(c Cache, src, dst string) (err error) {
 	src = filepath.Clean(src)
 	src, err = filepath.Abs(src)
 	if err != nil {
-		return err
+		return errors.Wrapf(err, "absolute path failed for %s", src)
```

Comment:
please check the vendor folder, the github.com/pkg/errors is already at master from latest merge, if you suggest not to use, then I'll change to fmt, thanks